packages/config/config/devices/0x0234/zdb5400.json

```diff
@@ -0,0 +1,556 @@
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Multi-Level Report, Pushbutton 1",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Basic Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Binary Switch Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"7": {
+			"label": "Multilevel Report, Pushbutton 2",
+			"maxNodes": 5
+		},
+		"8": {
+			"label": "Basic Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"9": {
+			"label": "Binary Switch Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"10": {
+			"label": "Multilevel Report, Pushbutton 3",
+			"maxNodes": 5
+		},
+		"11": {
+			"label": "Basic Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"12": {
+			"label": "Binary Switch Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"13": {
+			"label": "Multilevel Report, Pushbutton 4",
+			"maxNodes": 5
+		},
+		"14": {
+			"label": "Dimmer Basic Report",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Pushbutton 1"
+		},
+		"1[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Pushbutton 2"
+		},
+		"1[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Pushbutton 3"
+		},
+		"1[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Pushbutton 4"
+		},
+		"2": {
+			"$import": "templates/logic_group_template.json#dimming_duration"
+		},
+		"3": {
+			"$import": "templates/logic_group_template.json#duration_on_off"
+		},
+		"4": {
+			"$import": "templates/logic_group_template.json#dimmer_mode"
+		},
+		"5": {
+			"$import": "templates/logic_group_template.json#dimmer_min"
+		},
+		"6": {
+			"$import": "templates/logic_group_template.json#dimmer_max"
+		},
+		"7": {
+			"$import": "templates/logic_group_template.json#enable_central_scene"
+		},
+		"8": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Double-Activation Functionality",
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Sets light to 100%",
+					"value": 1
+				}
+			]
+		},
+		"10": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Enhanced LED Control"
+		},
+		"11": {
+			"$import": "templates/logic_group_template.json#pushbutton_debounce",
+			"unit": "10 ms",
+			"defaultValue": 5
+		},
+		"12": {
+			"$import": "../templates/master_template.json#base_1-255_nounit",
+			"label": "Pushbutton Press Threshold Time",
+			"unit": "10 ms",
+			"defaultValue": 20
+		},
+		"13": {
+			"$import": "../templates/master_template.json#base_1-255_nounit",
+			"label": "Pushbutton Held Threshold Time.",
+			"unit": "10 ms",
+			"defaultValue": 50
+		},
+		"14[0xff000000]": {
+			"$import": "templates/logic_group_template.json#global_brightness",
+			"label": "Global Brightness Red LED"
```

Comment:
Since this applies to all, lets just call this Brightness. "LED Indicator Brightness: Red" Etc.